build/python/backend/requirements.txt

```diff
@@ -8,6 +8,7 @@ cryptography==3.4.7
 docker==5.0.0
 eml-parser>=1.17
 esprima==4.0.1
+flare-capa==4.0.1
```

Comment:
Are we using `flare-capa` in this PR?

Reply:
Yes. We switched from the binary release (which I think is just a python binary wrapper) to the python module release. It's still called from subprocess.POpen very similarly though. I was attempting to close #159 and used [Method #2](https://github.com/mandiant/capa/blob/master/doc/installation.md#method-2-using-capa-as-a-python-library) for installation.

It turns out that capa isn't really built to be imported as a module into a script, the [main script](https://github.com/mandiant/capa/blob/master/capa/main.py) is a bit of a monolith and not well suited for our purposes.